plugins/helm/src/main/scala/org/broadinstitute/monster/sbt/MonsterHelmPlugin.scala

```diff
@@ -45,6 +46,19 @@ object MonsterHelmPlugin extends AutoPlugin {
       parsedMetadata.deepMerge(Json.obj("version" -> realVersion, "appVersion" -> realVersion))
     IO.write(tmpDir / "Chart.yaml", updatedMetadata.asYaml.spaces2)
 
+    // Inject the version into the values.yaml.
+    val chartValues = chartRoot / "values.yaml"
+    val parsedValues = if (chartValues.exists()) {
+      yaml.parser.parse(IO.read(chartValues)) match {
+        case Right(parsed) => parsed
+        case Left(err)     => sys.error(s"Could not parse $chartValues as YAML: ${err.getMessage}")
+      }
+    } else {
+      Json.obj()
```

Comment:
Is this defaulting to a an empty json object when a chartValue does not exist?

Reply:
This is defaulting for the case when `values.yaml` doesn't exist. Values within a chart sets the default values for parameters, so it's believable that we might hit a case where there are no default values.